pkg/operator/standalone_starter.go

```diff
@@ -33,6 +35,31 @@ func startControllerStandAlone(ctx context.Context, controllerConfig *controller
 	versionGetter := status.NewVersionGetter()
 	versionGetter.SetVersion("operator", status.VersionForOperatorFromEnv())
 
+	desiredVersion := status.VersionForOperatorFromEnv()
+	missingVersion := "0.0.1-snapshot"
+
+	// By default, this will exit(0) the process if the featuregates ever change to a different set of values.
+	featureGateAccessor := featuregates.NewFeatureGateAccess(
+		desiredVersion, missingVersion,
+		clients.ConfigInformers.Config().V1().ClusterVersions(), clients.ConfigInformers.Config().V1().FeatureGates(),
+		controllerConfig.EventRecorder,
+	)
```

Comment:
Will `exit(0)` trigger a new leader election? That will slow down feature enablement considerably, without any indication that the operator is Progressing.